src/lib/Libecl/ecl_verify_values.c

```diff
@@ -741,6 +741,9 @@ verify_value_mailpoints(int batch_request, int parent_object, int cmd,
 	if (strlen(pattr->value) == 0)
 		return PBSE_BADATVAL;
 
+	if (strlen(pattr->value) == 1 && *pattr->value == 'j')
+		return PBSE_BADATVAL;
+
 	if (strcmp(pattr->value, "n") != 0) {
 		pc = pattr->value;
 		while (*pc) {
```

Comment:
While you're making changes to this area, the loop is better suited as "for" than "while".

Reply:
Done